src/lib/init.js

```diff
@@ -49,10 +49,25 @@ export function init(configUpdates) {
 
 				// set cookies on digitrust domain after consent submitted
 				const { addEventListener, getVendorConsents } = cmp.commands;
+				if (config.digitrust.pixel) {
+					addEventListener('consentStringUpdated', digitrustPixel);
+				}
 				if (config.digitrust.redirects) {
 					addEventListener('consentStringUpdated', digitrustRedirect);
 				}
 
+				function digitrustPixel() {
+					getVendorConsents([64], (result) => {
+						if (result && result.vendorConsents && result.vendorConsents[64]) {
+							let iframe = document.createElement('iframe');
```

Comment:
Let's move the creation of an iframe to a separate utility function.

Reply:
moved into utils.js